src/cnxml/deserialize.js

```diff
@@ -77,7 +77,7 @@ const INLINE_TAGS = [
 const text = type => (el, next) => ({
     type: type,
     data: {
-        class: el.getAttribute('class') || '',
+        class: loadClasses(el),
     },
```

Comment:
We don't support classes on text elements. And while loading them is not a big issue, as they will be removed by schema, there's also no reason to do so.

Reply:
1. What makes titles different from other types of text (paragraphs) that they get to have classes?

2. Could you please stop marking conversations as resolved? It is _not your call_ to say that _my_ concerns were addressed.